compiler/sigmatch.nim

```diff
@@ -350,6 +350,14 @@ proc handleFloatRange(f, a: PType): TTypeRelation =
       else: result = isIntConv
     else: result = isNone
 
+template genericParamPut(c: var TCandidate; last, fGenericOrigin: PType) =
```

Comment:
This should be a proc, if possible. Code bloat is always a concern for rarely executed code.